src/ws/mock-config/cockpit/cockpit-alt.conf

```diff
@@ -4,7 +4,7 @@ ProtocolHeader = X-Forwarded-Proto
 [Ssh-Login]
 command = mock-auth-command
 host = default-host
-allowUnknown = true
+connectToUnknownHosts = true
```

Comment:
We should continue to test the old config name and environment variable name if we have code to support it. So lets copy this and create additional tests.